eng/config/PublishData.json

```diff
@@ -230,6 +230,20 @@
       "vsBranch": "main",
       "vsMajorVersion": 16
     },
+    "release/dev17.0-vs-deps": {
+      "nugetKind": [
+        "Shipping",
+        "NonShipping"
+      ],
+      "version": "4.0.*",
+      "packageFeeds": "default",
+      "channels": [
+        "dev17.0",
+        "dev17.0p1"
+      ],
+      "vsBranch": "main",
```

Comment:
not sure how this is used, but the target branch for insertion is feature/d17intial